CHANGES.md

```diff
@@ -1,3 +1,6 @@
+## Vaadin Core Elements 0.3.0-beta... (...)
+- Added Core word, so as the bundle includes all the set of Vaadin Components licensed as Apache 2.0.
```

Comment:
Also, why do we still have a CHANGES.md file? The same should apply here as with the elements, that we only have the release notes in GitHub.